packages/react-hooks/src/QuestionAnswerProvider.tsx

```diff
@@ -0,0 +1,104 @@
+import type { Agent } from '@aries-framework/core'
+import type {
+  QuestionAnswerRecord,
+  QuestionAnswerStateChangedEvent,
+  QuestionAnswerEventTypes,
+} from '@aries-framework/question-answer'
```

Comment:
Adding it as a dev dependency but importing the type, will cause issues if a project doesn't have the `@aries-framework/question-answer` package installed I think. Have you tested this in a proejct without QA installed?

Reply:
If we only import the types from the module it doesn't totally error out if the module isn't there, it just doesn't do anything when TypeScript resolves to JavaScript.